apps/api/lib/api/web/controllers/user_controller.ex

```diff
@@ -5,7 +5,7 @@ defmodule API.Web.UserController do
 
   def search(conn, %{"q" => query}) do
     current_user = Guardian.Plug.current_resource(conn)
-    users = API.User.search_users(query, except: current_user)
+    users = API.User.search(query, current_user.id)
```

Comment:
Just realized my benchmarking was flawed. Need to benchmark filtering at the db layer vs in memory again.

Reply:
@camirmas so it is ridiculously faster to filter in the db the way you did it. I'll revert that back later today in this pr.

`search username/email         100   10695.87 µs/op
search username/email         100   10720.94 µs/op
search username/email         100   10671.09 µs/op
search username/email         100   10561.55 µs/op
search username/email         100   10612.55 µs/op
******************************************
search username/email         500   6247.75 µs/op
search username/email         500   6430.57 µs/op
search username/email         500   6388.89 µs/op
search username/email         500   6401.46 µs/op
search username/email         500   6634.23 µs/op`